### PR TITLE
Fix/critical prod

### PR DESCRIPTION
**Pull request by hozzjss**
This fixes 3 critical bugs:
1. txid returned by proxy is no longer a stringified string.
2. emily url is not user editable any longer only set by the env
3. emily api key is no longer used or passed to emily for open apis

(edit by aldur) plus it makes it possible to set the minimum through an env variable

### CODE REVIEW

**Comment by aldur** [comment]
Fixes:

- https://github.com/stacks-network/sbtc/issues/1139
- https://github.com/stacks-network/sbtc/issues/1138

**Review by aldur — COMMENTED** [review]
Has this been tested again main? I'd wait to do that _before_ merging.

File: src/comps/Deposit.tsx [hunk 1]
```diff
@@ -306,9 +307,10 @@ const DepositFlowConfirm = ({
         bitcoinTxOutputIndex: 0,
         reclaimScript: reclaimScriptHex,
         depositScript: depositScriptHexPreHash,
-        url: emilyUrl,
       };
 
+      console.log({ emilyReqPayloadClient: emilyReqPayload });
+
```
Comment (matteojug) [inline src/comps/Deposit.tsx:313]: nit: can we also log this (with empty `bitcoinTxid`) before calling leather/xverse? For the current issue we are having we would not get any log

Comment (hozzjss) [reply src/comps/Deposit.tsx:329]: done

**Comment by hozzjss** [comment]
Fixes:
- https://github.com/stacks-network/sbtc/issues/1137
- https://github.com/stacks-network/sbtc/issues/1140
- https://github.com/stacks-network/sbtc/issues/1141

**Comment by aldur** [comment]
We have tested `754e1c5bf4b77762e462bc93cb1f713a2624acd3` on mainnet bridge. I would merge this PR up to that point and ask you to open a new PR for the remaining commits.

**Pull request merged**